apps/server/src/modules/common-cartridge/controller/common-cartridge.controller.ts

```diff
@@ -13,4 +14,10 @@ export class CommonCartridgeController {
 	public async exportCourse(@Param() exportCourseParams: ExportCourseParams): Promise<CourseExportBodyResponse> {
 		return this.commonCartridgeUC.exportCourse(exportCourseParams.parentId);
 	}
+
+	@Get('export/cards/:parentId')
+	public async exportCards(@Param() exportCourseParams: ExportCourseParams): Promise<CardListResponseDto> {
+		const ids: Array<string> = new Array<string>(exportCourseParams.parentId);

```

Comment:
a simple string[] for straightforward cases where you just need a list of strings and preform standard array operations, is a better choice.